File: src/main/java/ink/pd2/shell/plugin/Resources.java
package ink.pd2.shell.plugin;

import ink.pd2.shell.core.Command;
import ink.pd2.shell.core.Mark;

import java.util.Set;

public final class Resources {
    public static String getString(String key) {
        return Mark.INS.update(ink.pd2.shell.core.
                Resources.INS.getString(key));
    }
    public static void putString(String key, String value) {
        ink.pd2.shell.core.
                Resources.INS.putString(key, value);
    }
    public static String removeString(String key) {
        return ink.pd2.shell.core.
                Resources.INS.removeString(key);
    }

}

File: src/ink/pd2/shell/ui/Resources.kt
package ink.pd2.shell.ui

import ink.pd2.shell.log.writeLog

object Resources {
    //字符串
    private val strings = HashMap<String, String>(mapOf())
    fun getString(key: String): String {
        return strings[key] ?: return ""
    }
    fun putString(key: String, value: String) {
        strings[key] = value
        writeLog("Resources", "$key -> $value")
    }
    fun removeString(key: String): String? {
        return strings.remove(key)
    }
}
